optuna_dashboard/ts/types/index.d.ts

```diff
@@ -206,3 +208,11 @@ type StudyDetails = {
 type StudyParamImportance = {
   [study_id: string]: ParamImportance[][]
 }
+
+type PreferenceChoice = {
```

Comment:
nit: I think the name should contains a word "history".

```suggestion
type PreferenceHistoryChoice = {
```